title:Exception for FolderView

The following exception occurs at
"          List files = Arrays.asList(dir.listFiles(new FileFilter() {"
in FolderLoader.java class
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:304)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
at java.util.concurrent.FutureTask.run(FutureTask.java:242)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NullPointerException: storage == null
at java.util.Arrays$ArrayList.(Arrays.java:38)
at java.util.Arrays.asList(Arrays.java:155)
at com.naman14.timber.dataloaders.FolderLoader.getMediaFiles(FolderLoader.java:34)
at com.naman14.timber.adapters.FolderAdapter.updateDataSet(FolderAdapter.java:116)
at com.naman14.timber.adapters.FolderAdapter.(FolderAdapter.java:63)
at com.naman14.timber.fragments.FoldersFragmentMain$loadFolders.doInBackground(FoldersFragmentMain.java:100)
at com.naman14.timber.fragments.FoldersFragmentMain$loadFolders.doInBackground(FoldersFragmentMain.java:94)
at android.os.AsyncTask$2.call(AsyncTask.java:292)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
listFiles only returns null if specified file is not directory or there is an I/O exception. I have added a check for directory before calling list files
listFiles only returns null if specified file is not directory or there is an I/O exception. I have added a check for directory before calling list files